shared/enry.go

```diff
@@ -136,10 +137,20 @@ func GetColor(language string) string {
 	return enry.GetColor(language)
 }
 
+//export IsTest
+func IsTest(path string) bool {
+	return enry.IsTest(path)
+}
+
+//export GetLanguageType
+func GetLanguageType(language string) string {
+	return data.Type(enry.GetLanguageType(language)).String()
+}
+
 func strSliceCopy(result *[]*C.char, slice []string) {
 	for _, str := range slice {
 		*result = append(*result, C.CString(str))
 	}
 }
 
-func main() {}
+func main() {}
```

Comment:
This is a nit-pick, but would you mind [adding a newline](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file) so that GH does not complain?

Reply:
Sure